types/global.d.ts

```diff
@@ -0,0 +1,6 @@
+// Types for compiled templates
+declare module 'ember-boilerplate/templates/*' {
```

Comment:
On ne devrait pas avoir un fichier dédié pour ça ?

Reply:
ember-cli-typescript created this file so I left it as is, I think it’s better to stick to the blueprint!